Lib/platform.py

```diff
@@ -189,22 +189,28 @@ def libc_ver(executable=None, lib='', version='', chunksize=16384):
             # sys.executable is not set.
             return lib, version
 
-    libc_search = re.compile(b'(__libc_init)'
-                          b'|'
-                          b'(GLIBC_([0-9.]+))'
-                          b'|'
-                          br'(libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)', re.ASCII)
+    libc_search = re.compile(
+        br'(__libc_init)'
+        br'|'
+        br'(GLIBC_([0-9.]+))'
+        br'|'
+        br'(libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)'
+        br'|'
+        br'(musl-([0-9.]+))'
+        br'',
+        re.ASCII)
```

Comment:
BTW, we can switch to the VERBOSE mode, which would be slightly more clear.
```suggestion
    libc_search = re.compile(br'''
          (__libc_init)
        | (GLIBC_([0-9.]+))
        | (libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)
        | (musl-([0-9.]+))
        ''',
        re.ASCII | re.VERBOSE)
```

Reply:
I like that.